frontend/lib/src/util/utils.ts

```diff
@@ -343,15 +346,20 @@ export function setCookie(
   document.cookie = `${name}=${value};${expirationStr}path=/`
 }
 
-/** Return an Element's widget ID if it's a widget, and undefined otherwise. */
-export function getElementWidgetID(element: Element): string | undefined {
-  // NOTE: This is a temporary fix until the selections in maps work is done.
-  // We believe that this will be easier to fix when we get to that point so in
-  // the meantime we will be doing this simple fix to prevent this error: https://github.com/streamlit/streamlit/issues/8329
-  if (notNull(element.deckGlJsonChart)) {
```

Comment:
The new check for the id prefix is already excluding the id value of the Deckgl chart (since it is not a proper element ID)